engines/account/config/routes.rb

```diff
@@ -3,9 +3,9 @@
     namespace :v1 do
       post '/signups', to: 'signups#create'
       get '/signups/:token', to: 'signups#show'
+      post '/signups/:token', to: 'signups#complete'
```

Comment:
we are already breaking REST here, so if you prefer `/signups/complete/:token` I can change it